### PR TITLE
Fix bug 1238687 - Decommission Privacy Tips page

### PR DESCRIPTION
**Pull request by craigcook**
Redirect to SmartOn landing

### CODE REVIEW

**Comment by alexgibson** [comment]
Looks like there are still a couple of references to the page that could do with cleaning up:

* [Promo on the home page](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/home/home.html#L156) will throw a 500 error for locales that still show it.
* [Manifesto page](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/about/manifesto.html#L144) also throws a 500 error.
* Should be pretty safe to remove the link on the [default /whatsnew](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/australis/whatsnew.html#L61) template also, or alternatively point it to a new URL (people do still hit this fallback page, although traffic is relatively small).

File: bedrock/mozorg/redirects.py [hunk 1]
```diff
@@ -176,8 +176,12 @@ def decider(request, **kwargs):
     redirect(r'^m/privacy.html$', 'privacy.notices.firefox'),
 
     # Bug 1109318 /privacy/you -> privacy/tips/
+    # Bug 1238687 /privacy/tips -> teach/smarton/
     redirect(r'^privacy/you/?$',
-             'privacy.privacy-day'),
+             'teach.smarton.index'),
+    redirect(r'^privacy/tips/?$',
```
Comment (alexgibson) [inline bedrock/mozorg/redirects.py]: Should we add a [test for this redirect](https://github.com/mozilla/bedrock/blob/master/test_redirects/README.md)?

**Comment by alexgibson** [comment]
Aside from a bit of tidy up, change looks good r+wc :panda_face:

**Comment by craigcook** [comment]
Updated those URL references and added tests (hope I did that right).

File: bedrock/mozorg/templates/mozorg/about/manifesto.html [hunk 2]
```diff
@@ -141,8 +141,8 @@ <h1 itemprop="name">{{ self.page_title() }}</h1>
                 <section>
                   <h4>{{ _('Learn more') }}</h4>
                   <ul class="resources">
-                    <li><a href="{{ url('privacy.privacy-day') }}">{{ _('See how Mozilla works to put your privacy first') }}</a></li>
-                    <li><a href="https://blog.mozilla.org/privacy/">{{ _('Read about developments in privacy and data safety') }}</a></li>
+                    <li><a href="{{ url('teach.smarton.index') }}">{{ _('See how Mozilla works to put your privacy first') }}</a></li>
+                    <li><a href="https://blog.mozilla.org/netpolicy/">{{ _('Read about developments in privacy and data safety') }}</a></li>
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/about/manifesto.html]: Privacy blog is being absorbed by the net policy blog (https://bugzilla.mozilla.org/show_bug.cgi?id=1205091). The old URL will redirect, of course, but might as well update it here while I'm in the neighborhood.

**Comment by alexgibson** [comment]
LGTM, r+! :cake:

**Pull request merged**